dfastmi/kernel/core.py

```diff
@@ -244,10 +212,9 @@ def estimate_sedimentation_length2(
     L : float
         The expected yearly impacted sedimentation length [m].
     """
-    Lt = [tmi[i] * celerity[i] for i in range(len(tmi))]
+    yearly_sedimentation_lengths = [tmi[i] * celerity[i] for i in range(len(tmi))]
     
-    return sum(Lt) * 1000
-
+    return sum(yearly_sedimentation_lengths) * 1000
```

Comment:
Let's introduce a constant `km_to_m` equal to 1000 to show where it's coming from.

Reply:
I will add this, I will use capitals, since constants should be capital in python I think.